homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -191,6 +199,45 @@ async def async_step_machine_selection(
             errors=errors,
         )
 
+    async def async_step_bluetooth_selection(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle Bluetooth device selection."""
```

Comment:
This has nothing to do with reconfiguration of the config entry to do.
Please add this in a preliminary PR instead first.

Reply:
this has only to do with the reconfiguration of the config entry, as it's only used there

Reply:
I don't see the point of adding a step in a preliminary PR that is not used

Reply:
That seems very strange. Why would you have a bluetooth select in reconfigure but not in the normal setup then?
It was not meant to give additional configuration options in `reconfigure`

Reply:
Please leave it unresolved and in draft until the topic is resolved. Thanks

Reply:
>  It was not meant to give additional configuration options in `reconfigure`

Imo this is one of the main benefits of reconfigure, to give the user options for further configuration, that we leave out during inital setup for simplicity.

Reply:
No, because the it should be an `OptionsFlow` if it's optional settings that were not needed for the initial configuration.
`reconfigure` is meant to be more or less a copy of the original setup (in respect of the schema) to be able to reconfigure things that is mandatory to run the integration (which was maybe incorrectly setup first time or by other means necessary to change).

Reply:
> because normally it configures BT automatically, but I want to give the user the option to overwrite it during reconfguration

Why would this not be needed during the initial setup then?

Reply:
> Why would this not be needed during the initial setup then?

because it's unnecessary, config entry updates itself automatically on discovery in `init`

> which was maybe incorrectly setup first time or by other means necessary to change

that's exactly why I put it in reconfigure, in the unlikely case it fails it goes wrong